### PR TITLE
Create portuguese.json

### PR DESCRIPTION
**Pull request by pedrobonfim**
[Cantonese cheat sheet] (https://twitter.com/duckduckgo/status/687794472927768576) Inspired me to contribute with a Portuguese version.

What do you think?

https://duck.co/ia/view/portuguese_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@pedrobonfim Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub username:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/language/portuguese.json [hunk 1]
```diff
@@ -0,0 +1,175 @@
+{
+    "id": "portuguese_cheat_sheet",
+    "name": "Portuguese Cheat Sheet",
+    "description": "Basic Portuguese words and phrases",
+    "metadata": {
+        "sourceName": "dummies",
+        "sourceUrl": "http://www.dummies.com/how-to/content/portuguese-for-dummies-cheat-sheet.html"
+    },
+    "aliases": ["portuguese phrases", "english to portuguese", "basic portuguese phrases", "basic portuguese"],
+    "template_type": "reference",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/portuguese.json]: @pedrobonfim I think the template for languges is now "language"
@moollaza Isnt it?

**Comment by jophab** [comment]
@pedrobonfim Welcome to Duck Duck Hack :smile:  
Good job :+1:

File: share/goodie/cheat_sheets/json/language/portuguese.json [hunk 2]
```diff
@@ -0,0 +1,175 @@
+{
+    "id": "portuguese_cheat_sheet",
+    "name": "Portuguese Cheat Sheet",
+    "description": "Basic Portuguese words and phrases",
+    "metadata": {
+        "sourceName": "dummies",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/portuguese.json]: @pedrobonfim would look better if d is capitalised.

File: share/goodie/cheat_sheets/json/language/portuguese.json (hunk 1 above)
Comment (moollaza) [inline share/goodie/cheat_sheets/json/language/portuguese.json]: yes, but it's only necessary for Language templates with transliteration. You should still be able to use it though, you just won't see any difference (as expected).

**Comment by moollaza** [comment]
@pedrobonfim Thanks for contributing and welcome to DuckDuckHack!

Have you created an Instant Answer page for this Instant Answer? If so, please be sure to put a link to it in your description above.

> Details: http://docs.duckduckhack.com/submitting/pull-request.html

**Comment by pedrobonfim** [comment]
@moollaza Thanks, I have changed my pull request description!

**Comment by moollaza** [comment]
@pedrobonfim this should be deployed to beta.duckduckgo.com soon for further testing.

Do you happen to be a native Portuguese speaker? If not, I'd like to announce this on Twitter to gather any feedback the community might have

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Portuguese Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/portuguese_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Portuguese Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/portuguese_cheat_sheet_mobile.png)

**Comment by pedrobonfim** [comment]
@moollaza yes, I'm a native portuguese!

**Comment by moollaza** [comment]
@pedrobonfim this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by alohaas** [comment]
@pedrobonfim, thanks again for creating the Portuguese Cheat Sheet! You’ve done a really good job here and we'd like you to help our community tackle the next big challenge – our mission to become the best search engine for programmers: https://duck.co/blog/post/297/help-for-programmers.

When you're ready to move on, please: 

- Create a new Instant Answer to increase our Programming IA coverage

OR

- Test live programming Instant Answers (or those in development) for bugs and suggest improvements 

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Programming-IA-Coverage).

Thanks again!

**Pull request merged**